Gems/PhysX/Code/Source/BaseColliderComponent.cpp

```diff
@@ -64,13 +64,20 @@ namespace PhysX
 
             auto pxShape = static_cast<physx::PxShape*>(shapes[0]->GetNativePointer());
             physx::PxTransform pxWorldTransform = PxMathConvert(m_worldTransform);
-            physx::PxBounds3 bounds = physx::PxGeometryQuery::getWorldBounds(pxShape->getGeometry().any(),
+
+#if (PX_PHYSICS_VERSION_MAJOR == 5)
+            const physx::PxGeometry& pxShapeGeom = pxShape->getGeometry();
+#else
+            const physx::PxGeometry& pxShapeGeom = pxShape->getGeometry().any();
+#endif
```

Comment:
nit: I believe we generally like to include closing comments for defines (so this would have `#endif // PX_PHYSICS_VERSION_MAJOR == 5`)

Reply:
This is not needed for simple cases where the `#endif` is close to the `#if`: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#comments-after-preprocessor-statements